### DISCUSSION TITLE
Problem with OpenCV and Jupyterhub

### DISCUSSION THREAD

**Discussion by Iongng198** [community/community#31438] [New to GitHub]
Hi everyone,

I am new in Python and object Detection. I have followed the instruction about object detection and yolov4 [https://github.com/techzizou/yolov4-custom_Training](url). I applied everything same on Colab and it was working good. However, when I change the datasource and parameter to fit to my project. I am running on Jupyterhub server docker and got the problem with OpenCV and build darknet. 
![image](https://user-images.githubusercontent.com/112531600/187586468-68501898-1f41-4273-ab37-7c5d0c876e29.png)

OpenCV is already installed on Jupyterhub.

`Package                          Version
-------------------------------- ---------------------
oauth2client                     4.1.3
oauthlib                         3.1.1
object-detection                 0.1
opencv-python                    4.5.4.60
opencv-python-headless           4.6.0.66
openpyxl                         3.0.10
`

Have anyone know the solution for this?
Thank you so much.

**Comment by jfullstackdev** [comment]
I'm not very sure of that and not using OpenCV and Jupyterhub but what I'm seeing is when you change your folder name, that will make some directories invalid

it's just like packages in Java, you can't simply change the filenames without changing the package itself
